docs/pages/database-access/guides/redis-aws.mdx

```diff
@@ -0,0 +1,235 @@
+
+- Install Teleport `(=teleport.version=)`.
+- Set up Teleport to access your ElastiCache clusters for Redis.
+- Connect to your clusters through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![Teleport Database Access RDS Self-Hosted](../../../img/database-access/guides/redis_elasticache_selfhosted.png)
+</ScopedBlock>
+
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access RDS Cloud](../../../img/database-access/guides/redis_elasticache_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- AWS account with ElastiCache Redis clusters **with in-transit encryption
+  (TLS) enabled**.
+- Permissions to create and attach IAM policies.
+- A host, e.g., an EC2 instance, where you will run the Teleport Database
+  Service.
+
+## Step 1/7. Install Teleport
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+## Step 2/7. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+## Step 3/7. Create a Database Service configuration
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+Create the Database Service configuration:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ teleport db configure create \
+   -o file \
+   --proxy=teleport.example.com:3080 \
+   --token=/tmp/token \
+   --elasticache-discovery=us-west-1
+```
+
+</ScopedBlock>
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ teleport db configure create \
+   -o file \
+   --proxy=mytenant.teleport.sh \
+   --token=/tmp/token \
+   --elasticache-discovery=us-west-1
+```
+
+</ScopedBlock>
+
+The command will generate a Database Service configuration with ElastiCache
+database auto-discovery enabled on the `us-west-1` region and place it at the
+`/etc/teleport.yaml` location. See [Configuration
+Reference](../reference/configuration.mdx) for more details if you wish to
+customize the configuration.
+
+## Step 4/7. Create an IAM policy for Teleport
+
+Teleport needs AWS IAM permissions to be able to:
+
+- Discover and register ElastiCache Redis clusters.
+- Modify ElastiCache user passwords for Teleport managed users and
+  save the passwords in AWS Secrets Manager. 
+
+(!docs/pages/includes/database-access/aws-bootstrap.mdx!)
+
+## Step 5/7. Start the Database Service
+
+Start the Database Service:
+
+```code
+$ teleport start --config=/etc/teleport.yaml
+```
+
+The Database Service will discover and register all ElastiCache Redis clusters
+according to the configuration.
+
+<Admonition type="note" title="AWS credentials">
+  The Teleport Database Service uses the default credential provider chain to
+  find AWS credentials. See [Specifying
+  Credentials](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials)
+  for more information.
+</Admonition>
+
+## Step 6/7. Create a Teleport managed ElastiCache user (optional)
```

Comment:
When should readers ignore this step? If we're recommending that all readers follow this step, I think we should:

- Remove the "(optional)" parenthetical here
- In Step 7, use a single set of commands for connecting to the database instead of the `Tabs` component